README.md

```diff
@@ -1,28 +1,77 @@
-<!-- This is the readme template for commercetools nodejs projects. We recommend you following this format when writing a readme project -->
 [![commercetools logo][commercetools-icon]][commercetools]
 
-# Unslugified Project Title
+# CSV Parser Price
 [![Travis Build Status][travis-icon]][travis]
 [![Codecov Coverage Status][codecov-icon]][codecov]
 [![David Dependencies Status][david-icon]][david]
 [![David devDependencies Status][david-dev-icon]][david-dev]
 
-A short description of what this project is about and it's core features. This might be followed by some points to clarify:
-- Do something with this
-- Supports other stuff
+Convert [commercetools price](https://dev.commercetools.com/http-api-projects-products.html#price) CSV data to JSON.
 
 ## Usage
-How to use this project, possibly with a small code example. Could be split into CLI and direct usage.
+`npm install csv-parser-price`
+
 ### CLI
-...
+```
+Usage: csvparserprice [options]
+Convert commercetools price CSV data to JSON.
+
+Options:
+--help, -h        Show help text.                                    [boolean]
+--inputFile, -i   Path to CSV file.                         [default: "stdin"]
+--outputFile, -o  Input CSV file.                          [default: "stdout"]
+--batchSize, -b   Number of CSV rows to handle simultaneously.  [default: 100]
+--delimiter, -d   Used CSV delimiter.                           [default: ","]
+--strictMode, -s  Parse CSV strictly.                          [default: true]
+--projectKey, -p  API project key.                                  [required]
+--host            HTTP client host parameter.
+--protocol        HTTP client protocol parameter.
+--accessToken     HTTP client access token.
+```
 ### JS
-...
+```js
+const fs = require('fs');
+const CsvParserPrice = require('csv-parser-price');
+
+const apiCredentials = {
+    project_key: process.env.CM_PROJECT_KEY,
```

Comment:
Whoops that's a leftover, though it will work since it's just an environment variable.
Want to rename it to `SPHERE_PROJECT_KEY`?